ui/src/app/views/workspace/component.html

```diff
@@ -1,103 +1,16 @@
 <div class="workspace-container">
   <div class="main-body">
-    <h2>{{workspace.name}}</h2>
-    <div style="display: flex; justify-content: space-between">
-      <div style="margin-left: -0.5rem;">
-        <button class="btn btn-link" [class.selected]="tabOpen === Tabs.Notebooks" (click)="tabOpen = Tabs.Notebooks">
-          Notebooks ({{notebookList.length}})
-        </button>
-        <button class="btn btn-link final" [class.selected]="tabOpen === Tabs.Cohorts" (click)="tabOpen = Tabs.Cohorts">
-          Cohorts ({{cohortList.length}})
-        </button>
+    <h2 style="height: 65px">{{greeting}}</h2>
```

Comment:
Still need a decision on what to do if user does not have write permissions. Do we display cards greyed out, or not display at all? @lstrano-broad @fknabe

Reply:
I suspect we want greyed out? But defer to Lou and Fritz. 

I think we want this header to the right of the create card area? Just to check?

Reply:
From Insight it looks like that greeting is above the create card area